MobileCenterDistribute/MobileCenterDistribute/MSDistributeUtil.m

```diff
@@ -11,6 +12,12 @@
     NSString *mainBundlePath = [[NSBundle bundleForClass:[MSDistribute class]] resourcePath];
     NSString *frameworkBundlePath = [mainBundlePath stringByAppendingPathComponent:MOBILE_CENTER_DISTRIBUTE_BUNDLE];
     bundle = [NSBundle bundleWithPath:frameworkBundlePath];
+    
+    // Log to console in case the bundle is nil.
+    if(!bundle) {
+      MSLogWarning([MSDistribute logTag], @"The MobileCenterDistributeResources.bundle file could not be found in your"
```

Comment:
Good question. I was not sure. Thinking about it now, I'm changing this to error as the SDK wasn't integrated properly.